### ISSUE TITLE
rdfs:crossRef

### ISSUE THREAD

**Issue by justin2004** [spdx/LicenseListPublisher#94]
this property/predicate is generated by this tool:
```
<http://www.w3.org/2000/01/rdf-schema#crossRef>
```

see:
https://github.com/spdx/license-list-data/blob/6820cab01008b520a72289bbdc6b2a895afc2590/rdfturtle/0BSD.turtle#L8

but i don't think it is in the rdfs vocabulary:
https://www.w3.org/TR/rdf-schema/

should we declare 
```
<http://spdx.org/rdf/terms#crossRef>
```
instead and add a definition?

**Comment by goneall** [comment]
@justin2004 Good catch - clearly a bug and your solution is the correct one.  I'll work on a PR to correct this.

**Comment by goneall** [comment]
It looks like this has already been fixed - we just need to deploy the new utility to the LicenseListXML repo which is planned in the next week or so.

**Comment by goneall** [comment]
This is resolved with the 2.2.0 release of the LicenseListPublisher
